tests/system-tests/o-cloud/o_cloud_suite_test.go

```diff
@@ -0,0 +1,72 @@
+package o_cloud_system_test
+
+import (
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	. "github.com/openshift-kni/eco-gotests/tests/internal/inittools"
+	"github.com/openshift-kni/eco-gotests/tests/internal/reporter"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/o-cloud/internal/ocloudcommon"
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/o-cloud/internal/ocloudinittools"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/o-cloud/internal/ocloudparams"
+	_ "github.com/openshift-kni/eco-gotests/tests/system-tests/o-cloud/tests"
+)
+
+var (
+	_, currentFile, _, _ = runtime.Caller(0)
+)
+
+func TestOCloud(t *testing.T) {
+	_, reporterConfig := GinkgoConfiguration()
+	reporterConfig.JUnitReport = GeneralConfig.GetJunitReportPath(currentFile)
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "O-Cloud SystemTests Suite", Label(ocloudparams.Labels...), reporterConfig)
+}
+
+var _ = BeforeSuite(func() {
+	ocloudcommon.VerifyNamespaceExists(ocloudparams.AcmNamespace)
```

Comment:
imo we can assume the required operators were correctly installed at the time when the tests run so I don't think we should cover their installation validation in the tests.

Reply:
I have removed this validation